pkg/rpc/client/client.go

```diff
@@ -117,17 +117,17 @@ func (c *Client) Init() error {
 	}
 	c.network = version.Magic
 	c.stateRootInHeader = version.StateRootInHeader
-	neoContractHash, err := c.GetContractStateByAddressOrName("neo")
+	neoContractHash, err := c.GetContractStateByAddressOrName("NeoToken")
```

Comment:
We also need to change everything that connected with `(*Client).cache.nativeHashes`, there are still old names (look at the `(c *Client) Init()`).

Reply:
Fixed.

Reply:
One old name is left, let's fix it too, see `cli/wallet/wallet.go`, line 236: `neoContractHash, err := c.GetNativeContractHash("NeoToken")`.

Reply:
And let's remove `policyName` constant from `netive/policy.go` and use `nativenames.Policy` instead.